core/tests/data/test2_svg.svg

```diff
@@ -0,0 +1 @@
+<svg height="512pt" viewBox="0 0 512 512" width="512pt" xmlns="http://www.w3.org/2000/svg"><path d="m416 512h-320c-53.023438 0-96-42.976562-96-96v-320c0-53.023438 42.976562-96 96-96h320c53.023438 0 96 42.976562 96 96v320c0 53.023438-42.976562 96-96 96zm0 0" fill="#e3f8fa"/><path d="m333.328125 128h-154.671875c-16.175781 0-29.328125 13.152344-29.328125 29.328125v197.328125c0 16.175781 13.152344 29.328125 29.328125 29.328125h154.671875c16.175781 0 29.328125-13.152344 29.328125-29.328125v-197.328125c.015625-16.175781-13.152344-29.328125-29.328125-29.328125zm-130.65625 173.328125c12.351563 0 21.328125 8.222656 21.328125 19.535156 0 11.472657-9.375 20.464844-21.328125 20.464844h-13.328125c-4.414062 0-8-3.585937-8-8 0-4.417969 3.585938-8 8-8h13.328125c2.65625 0 5.328125-1.375 5.328125-4.464844 0-3.085937-3.34375-3.535156-5.328125-3.535156-12.351563 0-21.328125-8.335937-21.328125-19.839844 0-11.503906 9.167969-20.160156 21.328125-20.160156h13.328125c4.414062 0 8 3.582031 8 8 0 4.414063-3.585938 8-8 8h-13.328125c-.894531 0-5.328125.191406-5.328125 4.160156-.015625 3.566407 4.082031 3.839844 5.328125 3.839844zm74.367187-13.855469-13.328124 48c-.957032 3.457032-4.109376 5.855469-7.710938 5.855469s-6.753906-2.398437-7.710938-5.855469l-13.328124-48c-1.183594-4.257812 1.3125-8.671875 5.566406-9.855468 4.226562-1.183594 8.671875 1.328124 9.855468 5.566406l5.617188 20.273437 5.632812-20.257812c1.183594-4.238281 5.597657-6.734375 9.855469-5.566407 4.238281 1.183594 6.734375 5.582032 5.550781 9.839844zm29.632813 37.855469h8v-3.121094c-3.105469-1.085937-5.328125-4.0625-5.328125-7.535156 0-4.414063 3.585938-8 8-8h5.328125c4.417969 0 8 3.585937 8 8v18.671875c0 4.417969-3.582031 8-8 8h-16c-10.289063 0-18.671875-8.367188-18.671875-18.671875v-26.671875c0-10.289062 8.367188-18.671875 18.671875-18.671875h16c4.417969 0 8 3.582031 8 8 0 4.414063-3.582031 8-8 8h-16c-1.472656 0-2.671875 1.199219-2.671875 2.671875v26.671875c0 1.457031 1.199219 2.65625 2.671875 2.65625zm0 0" fill="#26c6da"/></svg>
```

Comment:
You might need a newline at the end of the file...

Reply:
Done.